packages/thirdweb/src/react/web/ui/ConnectWallet/Details.tsx

```diff
@@ -260,7 +260,7 @@ export const ConnectedWalletDetails: React.FC<{
             color="secondaryText"
             weight={400}
           >
-            {formatBalanceOnButton(Number(balanceQuery.data.displayValue))}
+            {formatBalanceOnButton(Number(balanceQuery.data.displayValue))}{" "}
             {balanceQuery.data.symbol}
```

Comment:
use string concat instead


```suggestion
            {`${formatBalanceOnButton(Number(balanceQuery.data.displayValue))} ${balanceQuery.data.symbol}`}
```